tests/common_setup.py

```diff
@@ -202,7 +202,7 @@ def fin():
 
 
 @pytest.fixture(scope="function")
-def multitenancy_setup_without_client(request):
+def enterprise_no_client(request):
     if not conftest.run_tenant_tests:
```

Comment:
What about changing also the conf parameter to something like `conftest.run_enterprise_tests`?

Reply:
we'll be getting back to this soon, don't want to touch too much at once.

we'll be introducing a new env var, maybe replacing this arg, or maybe doing something clever to it.